libs/ui/src/lib/terminal/terminal.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="command">$ {{ command }}</div>
+<div *ngIf="command" class="command">$ {{ command }}</div>
```

Comment:
why would not command be defined?

Reply:
It is not defined in the background task UI since it is displayed in the list of tasks itself